ui/tests/integration/components/horizontal-bar-chart-test.js

```diff
@@ -0,0 +1,26 @@
+import { module, test } from 'qunit';
+import { setupRenderingTest } from 'ember-qunit';
+import { render } from '@ember/test-helpers';
+import { hbs } from 'ember-cli-htmlbars';
+
+module('Integration | Component | horizontal-bar-chart', function(hooks) {
```

Comment:
I deleted my default test file, feel free to keep it but maybe make a note to add or amend to it.

Reply:
Ah yeah - I was wondering what to do with it... I'll add a note to fix